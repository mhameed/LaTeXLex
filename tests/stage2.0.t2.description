author: ehc
date: 09/04/08
description: to check that the right thing happens when we have blank lines inside an item and when there is a \item[something] 