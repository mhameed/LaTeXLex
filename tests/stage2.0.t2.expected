\begin{itemize}
\item This is the first item on one line 
\item This is the second item split across two lines 
\item This is the third item following on the same line by the fourth
\item This is the fourth item 
\begin{itemize}
\item[i]
This is the first item on one line 
\item[ii]
This is the second item split across two lines 
\item[iii] 
This is the third item following on the same line by the fourth
\item[iv] This is the fourth item
\end{itemize}
\end{itemize}

